src/components/TimeLog/TimeForm.js

```diff
@@ -8,19 +8,47 @@ class TimeForm extends Component {
     end: ""
   }
 
+  handleChangeStart = (e) => {
+    this.setState({
+      start: e.target.value
+    });
+  }
+
+  handleChangeEnd = (e) => {
+    this.setState({
+      end: e.target.value
+    });
+  }
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    let entry = {
+      start: parseTime(this.state.start),
+      end: parseTime(this.state.end)
+    }
+    this.props.onAdd(entry);
+    this.setState({
```

Comment:
One thing I like to do when I have to "reset state" is create a literal that's called `INITIAL_STATE` with the properties empty/null. When I setup my `state` object, like you are on line 6, you can pass INITIAL_STATE there, and then when I need to reset state, I can pass the _same_ object here. The result is the same, but I have a central point of configuration for what "empty" looks like and that means i'm less likely to forget to update one of these as features evolve.